src/cmd/package.go

```diff
@@ -308,15 +311,18 @@ var packagePullCmd = &cobra.Command{
 	Example: lang.CmdPackagePullExample,
 	Args:    cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
-		pkgConfig.PkgOpts.PackageSource = args[0]
-		pkgClient, err := packager.New(&pkgConfig)
+		outputDir := pkgConfig.PullOpts.OutputDirectory
+		if outputDir == "" {
+			wd, err := os.Getwd()
+			if err != nil {
+				return err
+			}
+			outputDir = wd
+		}
+		err := packager2.Fetch(cmd.Context(), args[0], outputDir, pkgConfig.PkgOpts.Shasum, filters.Empty())
```

Comment:
Very supportive of this change :)
However, I'm unsure if it's worth it to pull in the filters pkg to just pass in a `filters.Empty()` into this function. Could you expand on why this approach was chosen?

Reply:
I do this because mirror will need filter to work.